### PR TITLE
drivers replace math::Vector<3> with matrix::Vector3f

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

File: src/drivers/device/integrator.h [hunk 1]
```diff
@@ -84,7 +84,7 @@ class Integrator
 	 * @param integral_dt	Get the dt in us of the current integration (only if reset).
 	 * @return		the integral since the last read-reset
 	 */
-	math::Vector<3>		get(bool reset, uint64_t &integral_dt);
+	matrix::Vector3f		get(bool reset, uint64_t &integral_dt);
```
Comment (bkueng) [inline src/drivers/device/integrator.h]: Nitpick: spacing is off now

File: src/drivers/device/integrator.h [hunk 2]
```diff
@@ -95,18 +95,18 @@ class Integrator
 	 * @param filtered_val	The integral differentiated by the integration time.
 	 * @return		the integral since the last read-reset
 	 */
-	math::Vector<3>		get_and_filtered(bool reset, uint64_t &integral_dt, math::Vector<3> &filtered_val);
+	matrix::Vector3f		get_and_filtered(bool reset, uint64_t &integral_dt, matrix::Vector3f &filtered_val);
```
Comment (bkueng) [inline src/drivers/device/integrator.h]: spacing

File: src/drivers/device/integrator.h [hunk 3]
```diff
@@ -95,18 +95,18 @@ class Integrator
...
 	uint64_t _auto_reset_interval;			/**< the interval after which the content will be published
 							     and the integrator reset, 0 if no auto-reset */
 	uint64_t _last_integration_time;		/**< timestamp of the last integration step */
 	uint64_t _last_reset_time;			/**< last auto-announcement of integral value */
-	math::Vector<3> _alpha;				/**< integrated value before coning corrections are applied */
-	math::Vector<3> _last_alpha;			/**< previous value of _alpha */
-	math::Vector<3> _beta;				/**< accumulated coning corrections */
-	math::Vector<3> _last_val;			/**< previous input */
-	math::Vector<3> _last_delta_alpha;		/**< integral from previous previous sampling interval */
+	matrix::Vector3f _alpha;				/**< integrated value before coning corrections are applied */
```
Comment (bkueng) [inline src/drivers/device/integrator.h]: spacing

File: src/drivers/device/integrator.h (hunk 1 above)
Comment (dagar) [reply src/drivers/device/integrator.h]: fixed

File: src/drivers/device/integrator.h (hunk 2 above)
Comment (dagar) [reply src/drivers/device/integrator.h]: fixed

File: src/drivers/device/integrator.h (hunk 3 above)
Comment (dagar) [reply src/drivers/device/integrator.h]: fixed

**Pull request merged**